os/Kconfig

```diff
@@ -39,6 +39,20 @@ config DEFAULT_SMALL
 		have already been selected in your configuration file.  This applies
 		only to new settings that require a default value.
 
+config DOCKER_VERSION
+	string "Docker Image Version"
+	default "1.5.8"
```

Comment:
@sunghan-chang , I had a discussion with @abhishek-samsung, we want to keep "Docker Image Version" inside "Build Setup". Since LIBCXX_EXCEPTION is define in "Built-in Libraries" (below "Build Setup"), hence we can't use this variable for DOCKER_VERSION.